From 7f1e3dd7c80d4f56eaebb289de8794f6a29ff197 Mon Sep 17 00:00:00 2001
From: "Rob Herring (Arm)" <robh@kernel.org>
Date: Thu, 13 Aug 2026 12:39:58 -0500
Subject: [PATCH 14053/15333] dt-bindings: devfreq: samsung,exynos-ppmu: Use
 standard regex syntax

The "{,N}" regex syntax is non-standard, but happens to be supported by
python re module. It is equivalent to "{0,N}" which is more commonly
supported. This fixes using a rust implementation of the DT schema
tools.

Link: https://patch.msgid.link/20260813174037.2263884-1-robh@kernel.org
Signed-off-by: Rob Herring (Arm) <robh@kernel.org>
---
 .../devicetree/bindings/devfreq/event/samsung,exynos-ppmu.yaml  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/devfreq/event/samsung,exynos-ppmu.yaml b/Documentation/devicetree/bindings/devfreq/event/samsung,exynos-ppmu.yaml
index d27dcb2fef12..a3f0cd508b3a 100644
--- a/Documentation/devicetree/bindings/devfreq/event/samsung,exynos-ppmu.yaml
+++ b/Documentation/devicetree/bindings/devfreq/event/samsung,exynos-ppmu.yaml
@@ -40,7 +40,7 @@ properties:
     type: object
 
     patternProperties:
-      '^ppmu-event[0-9]+(-[a-z0-9]+){,2}$':
+      '^ppmu-event[0-9]+(-[a-z0-9]+){0,2}$':
         type: object
         properties:
           event-name:
-- 
2.55.0

